src/task/task.service.ts

```diff
@@ -1,40 +1,48 @@
 import { Injectable } from '@nestjs/common';
+import { PinoService } from '../logger';
 import { CreateTaskDto, UpdateTaskDto } from './dto';
 
 @Injectable()
 export class TaskService {
+  private readonly ctx = TaskService.name;
+
+  constructor(private readonly logger: PinoService) {}
```

Comment:
```suggestion
  private readonly logger = new Logger(TaskService.name);
```
При этом `Logger` импортнуть из неста - `core` или `common`, точно не помню